modules/monitoring-default-monitoring-components.adoc

```diff
@@ -26,6 +26,12 @@ By default, the {product-title} {product-version} monitoring stack includes thes
 |Prometheus Adapter
 |The Prometheus Adapter (PA in the preceding diagram) translates Kubernetes node and pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal pod autoscaling. The Prometheus Adapter is also used by the `oc adm top nodes` and `oc adm top pods` commands.
 
+|Metrics Server (Technology Preview)
+|If enabled, the Metrics Server component collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality.
+
+The Technology Preview of the Metrics Server component is automatically installed instead of Prometheus Adapter if you configure the `FeatureGate` custom resource with the `TechPreviewNoUpgrade` option.
```

Comment:
Just checking - do you know whether there should be an article before Prometheus Adapter?
```suggestion
The Technology Preview of the Metrics Server component is automatically installed instead of the Prometheus Adapter if you configure the `FeatureGate` custom resource with the `TechPreviewNoUpgrade` option.
```

Reply:
Good question! Currently, we don't use an article.